openapi/nytimes.moviereviews/client.bal

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import  ballerina/http;
+import  ballerina/url;
+import  ballerina/lang.'string;
+
+public type ApiKeysConfig record {
+    map<string> apiKeys;
+};
+
+# With the Movie Reviews API, you can search New York Times movie reviews by keyword and get lists of NYT Critics' Picks.
+#
+# + clientEp - Connector http endpoint
+public client class Client {
+    http:Client clientEp;
+    map<string> apiKeys;
+    # Client initialization.
+    #
+    # + apiKeyConfig - API key configuration detail
+    # + clientConfig - Client configuration details
+    # + serviceUrl - Connector server URL
+    # + return - Error at failure of client initialization
+    public isolated function init(ApiKeysConfig apiKeyConfig, http:ClientConfiguration clientConfig =  {}, string serviceUrl = "http://api.nytimes.com/svc/movies/v2") returns error? {
+        http:Client httpEp = check new (serviceUrl, clientConfig);
+        self.clientEp = httpEp;
+        self.apiKeys = apiKeyConfig.apiKeys;
+    }
+    # Get movie reviews that are critics' picks. You can either specify the reviewer name or use "all", "full-time", or "part-time".
+    #
+    # + 'resource\-type - all | full-time | part-time | [reviewer-name]
+    # + return - An array of Movie Critics
+    remote isolated function criticsPicks(string 'resource\-type) returns InlineResponse200|error {
```

Comment:
@aneeshafedo We have fixed this type of variable name `string 'resource\-type` as I remember. Could you please check what causing this?

Reply:
Raised an issue regarding this https://github.com/ballerina-platform/ballerina-openapi/issues/442
This was fixed in d5420ba by editing the yaml file.